edit/edit.js

```diff
@@ -1744,12 +1747,17 @@ function showKeyMapHelp() {
     const col = input.parentNode.cellIndex;
     inputs[1 - col].value = '';
     table.tBodies[0].childNodes.forEach(row => {
-      let cell = row.children[col];
-      cell.innerHTML = cell.textContent.replace(query, '<mark>$&</mark>');
+      const cell = row.children[col];
```

Comment:
Oops... indeed. Well, it can be implemented via string#split or regexp#exec. Someday you'll have to do it anyway (due to AMO) so it's up to you.